src/Accelerators/NNPA/Dialect/ZHigh/ZHighOps/Stick/ZHighStick.td

```diff
@@ -83,6 +83,38 @@ def ReplaceONNXLeakyReluPattern: Pat<
    (SameLayout $X, $stickout)]
 >;
 
+// The pattern
+//   zhigh.Stick (onnx.Softplus (zhigh.Unstick (%X)))
+// can be replaced by
+//   %minusOne = zhigh.Stick(GetConstantOfType<"-1.0">, %X)
+//   %minusX = zhigh.Mul(%X, %minusOne)
+//   zhigh.Add (
+//       zhigh.Relu(%X),
+//       zhigh.log(zhigh.Sub(zhigh.Exp(zhigh.Min(%X,%minusX)), %minusOne)))
+//
+// Constraints:
+//   - %X should have static shape
+//
```

Comment:
Could you please and a link to the website that explains how to derive `softplus(x) = max(0,x) + log(exp(-|x|) -1)`? I remember you show the link in a local meeting.

Also, please explain a bit about replacing `-|x|` by `min(x, -x)`. I guess the reason is NNPA does not have a function to compute `abs(x)`.

Reply:
Added reference websites and explanation about -|x| replacement. Thanks.